setup.cfg

```diff
@@ -15,22 +15,23 @@ classifiers =
   Environment :: Console
   Intended Audience :: Science/Research
   Operating System :: OS Independent
-  Programming Language :: Python :: 3.6
-  Programming Language :: Python :: 3.7
   Programming Language :: Python :: 3.8
   Programming Language :: Python :: 3.9
+  Programming Language :: Python :: 3.10
+  Programming Language :: Python :: 3.11
+  Programming Language :: Python :: 3.12
   Programming Language :: Python :: 3 :: Only
   Topic :: Utilities
 license_file = LICENSE
 long_description = file: README.md
 long_description_content_type = text/markdown
 
 [options]
-python_requires = >= 3.6
+python_requires = >= 3.8
```

Comment:
This seemed to be out of date with your stated python version support?

Reply:
Yeah probably just outdated docs